app/views/did_you_know_alert_mailer/email.html.haml

```diff
@@ -0,0 +1,20 @@
+%link{rel: 'stylesheet', href:'/mailer.css'}
+%p.paragraph
+  Hello,
+  %p.paragraph
+    An article that is being worked on as part of your course,
+    %a.link{href: @alert.article.url}= @alert.article.title
+    has received a Did You Know nomination.
+    %a.link{href: "https://upload.wikimedia.org/wikipedia/commons/e/ee/Classroom_handout_-_Submitting_an_article_to_the_Did_You_Know_process.pdf"}= "#{"See the PDF handout about the Did You Know process for more details."} "
```

Comment:
Too much of the text is part of the link here. I think just linking the "PDF handout about the Did You Know process" part works better.

It's also not necessary to use string interpolation here. If there are no variables involved, you can do
```haml
%a.link{href: foo}
  Link text
```

Reply:
yes ok